tests/ha/test_multi_server_pod_failure.py

```diff
@@ -629,3 +629,584 @@ def test_writes_till_kserver_pods_fail(self):
+        LOGGER.info("Step 3: Successfully shutdown %s (K) server pods one by one while "
+                    "continuous DELETEs in background", self.kvalue)
+        event.clear()
+
+        LOGGER.info("Step 4: Check cluster status")
+        resp = self.ha_obj.check_cluster_status(self.node_master_list[0])
+        assert_utils.assert_false(resp[0], resp)
+        LOGGER.info("Step 4: Cluster has some failures due to %s server pods has gone down.",
+                    self.kvalue)
+
+        LOGGER.info("Step 5: Check services status that were running on server "
+                    "pods which are deleted.")
+        counter = 0
+        for pod_name in self.pod_name_list:
+            hostname = self.pod_dict.get(pod_name)[0]
+            resp = self.hlth_master_list[0].get_pod_svc_status(pod_list=[pod_name], fail=True,
+                                                               hostname=hostname)
+            LOGGER.debug("Services status on %s : %s", pod_name, resp)
+            if not resp[0]:
+                counter += 1
+            pod_list.remove(pod_name)
+        assert_utils.assert_equal(counter, 0, "Services on some server pods not stopped.")
+        LOGGER.info("Step 5: Services of server pods are in offline state")
+
+        LOGGER.info("Step 6: Check services status on remaining pods %s", server_data_pod_list)
+        resp = self.hlth_master_list[0].get_pod_svc_status(pod_list=server_data_pod_list,
+                                                           fail=False)
+        LOGGER.debug("Response: %s", resp)
+        assert_utils.assert_true(resp[0], resp)
+        LOGGER.info("Step 6: Services of remaining pods are in online state")
+
+        LOGGER.info("Step 7: Verify status for In-flight DELETEs while %s (K) server pods were"
+                    "going down", self.kvalue)
+        LOGGER.info("Waiting for background DELETEs thread to join. Waiting for %s seconds to "
+                    "collect the queue logs", HA_CFG["common_params"]["60sec_delay"])
+        thread.join()
+        del_resp = tuple()
+        while len(del_resp) != 2:
+            del_resp = del_output.get(timeout=HA_CFG["common_params"]["60sec_delay"])
+        if not del_resp:
+            assert_utils.assert_true(False, "Background process failed to do deletes")
+        event_del_bkt = del_resp[0]
+        fail_del_bkt = del_resp[1]
+        rem_bkts_aftr_del = s3_test_obj.bucket_list()[1]
+        assert_utils.assert_false(len(fail_del_bkt),
+                                  f"Bucket deletion failed when cluster was online {fail_del_bkt}")
+        assert_utils.assert_true(len(rem_bkts_aftr_del) < del_bucket,
+                                 "Some bucket deletion expected during pods going down")
+        LOGGER.info("Step 7: Verified status for In-flight DELETEs while %s (K) server pods were"
+                    "going down", self.kvalue)
+
+        LOGGER.info("Step 8: Perform DELETEs on remaining FailedToDelete buckets "
```

Comment:
I think we should do this step in last, 
the remaining + failed to delete when server pod was going down as this test is specific to delete, we should try to deleted all the created 150 buckets as a part of this test only.

Reply:
if we delete all then READ will on which buckets? @ankita-93 
I guess step is to delete failed to delete and the read on remaining

Reply:
no change needed